proposals/lip-0014.md

```diff
@@ -506,9 +509,9 @@ The node chooses one peer `P` at random from the set `bestPeers` and request its
 
 **3. Step: Revert to last common block**
 
-  a)  The node requests the last common block `C` from `P`.
+  a)  The node requests the last common block `C` from `P`. If `P` does not provide a common block `C` after several tries, then `P` is banned and the block synchronization is restarted from Step 1.
```

Comment:
Should the number of tries be specified? or is it up to the node operator to choose the value?

Reply:
Good point. Our implementation will definitely have a set number for the amount of tries, but it would not break the protocol to use another number. Maybe we can make a non-mandatory recommandation ?

Reply:
I changed `several` to `three`, but this value can be changed in the implementation (if three is to harsh, for instance).